packages/core/content-releases/admin/src/pages/tests/Releases.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ReleasesPage } from '../Releases';
+
+const user = userEvent.setup();
+
+const render = () => ({
+  ...renderRTL(<ReleasesPage />, {
+    wrapper: ({ children }) => (
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}} defaultLocale="en">
+          {children}
+        </IntlProvider>
+      </ThemeProvider>
+    ),
+  }),
+});
+
+describe('Releases home page', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the Releases title as an heading', () => {
+    const { getByRole } = render();
+    const pageTitle = getByRole('heading', { level: 1 });
+    expect(pageTitle).toHaveTextContent('Releases');
+  });
+
+  it('shows a subtitle with the number of releases', () => {
+    const { getByText } = render();
+    // if there are 0 releases
+    expect(getByText('No releases')).toBeInTheDocument();
+  });
```

Comment:
these would be better as a single test. See this post from the RTL author: https://kentcdodds.com/blog/write-fewer-longer-tests